tutorials/setup-k8s-cluster-rancher/index.mdx

```diff
@@ -16,9 +16,9 @@ dates:
 
 ## Rancher Overview
 
-Rancher is an open-source container management platform, providing an graphical interface making container management easier.
+Rancher is an open-source container management platform, providing a graphical interface making container management easier.
 
-The GUI of Rancher makes it easy to manage secrets, handle roles and permissions. It allows to scale nodes and pods, set up load balancers without the requirement of a command line tool or the editing of hard to read YAML files.
+The GUI of Rancher makes it easy to manage secrets, handle roles and permissions. It allows scaling nodes and pods, set up load balancers without the requirement of a command line tool or the editing of hard to read YAML files.
```

Comment:
To me, a more natural-sounding correction for this structure (in other places too) would either be:
- It allows you to scale node and pods
OR
- It allows the scaling of nodes and pods

(also the other verbs later in the sentence then need to change form accordingly, either way)